docs/source/technical_tutorials/user_data/demo_memory_profiling.py

```diff
@@ -0,0 +1,17 @@
+from memory_profiler import profile
+
+"""
+... Code here
+"""
+
+@profile
+
```

Comment:
trivial: remove blank line after decorator -- Python convention is to have no blank lines between a decorator and the function it decorates